### PR TITLE
refactor: prefix css vars with anglify

### PR DESCRIPTION
**Pull request by iCrawl**

### CODE REVIEW

File: apps/docs/src/app/pages/component-pages/progress-linear-page/progress-linear-page.component.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
     height: 200px;
 
     anglify-progress-linear {
-      --progress-linear-height: var(--progress-linear-playground-size);
+      --anglify-progress-linear-height: var(--anglify-progress-linear-playground-size);
```
Comment (valentingavran) [inline apps/docs/src/app/pages/component-pages/progress-linear-page/progress-linear-page.component.scss:6]:
```suggestion
      --anglify-progress-linear-height: var(--docs-progress-linear-playground-size);
```

Comment (valentingavran) [reply apps/docs/src/app/pages/component-pages/progress-linear-page/progress-linear-page.component.scss:6]: `apps/docs/src/app/pages/component-pages/progress-linear-page/progress-linear-page.component.ts`:

```typescript
  @HostBinding('style.--docs-progress-linear-playground-size')
  public get getSizeInPx() {
  // ...
}
```

**Review by valentingavran — CHANGES_REQUESTED** [review]

File: apps/docs/src/app/pages/component-pages/progress-linear-page/progress-linear-page.component.scss (hunk 1 above)
Comment (iCrawl) [reply apps/docs/src/app/pages/component-pages/progress-linear-page/progress-linear-page.component.scss:6]: 😆 oops

**Review by valentingavran — APPROVED** [review]
⭐️

**Pull request merged**